docs.html

```diff
@@ -697,6 +698,100 @@ <h3>Javascript</h3>
+                    Curabitur blandit fermentum eleifend.
+                  </div>
+                </div>
+                <div class="card-footer-basic fluid-x">
+                  <div class="footer-links fluid-x">
+                    <button class="primary-btn-card">READ</button>
+                    <button class="primary-btn-card">BOOKMARK</button>
+                  </div>
+                  <div class="footer-icons fluid-x">
+                    <button class="icon-btn"><i class="far fa-heart"></i></button>
+                    <button class="icon-btn"><i class="fas fa-share-alt"></i></button>
+                    <button class="icon-btn"><i class="fas fa-ellipsis-v"></i></button>
+                  </div>
+                </div>
+                <div class="card-badge">
+                  In progress
+                </div>
+              </div>
+            </div>
+            <div class="grid-item-two">
+              <div class="card-container relative-pos">
+                <div class="card-image-basic">
+                  <img src="./images/maxim-berg-6-NP_CdNqtU-unsplash.jpg" alt="" />
+                </div>
+                <div class="card-typography fluid-y">
+                  <div class="padding-l-r-16-b-5 heading">3D render</div>
+                  <div class="padding-l-r-16-b-5 sub-heading">by Maxim Berg</div>
+                  <div class="card-body padding-l-r-16-b-5">
+                    Nunc consequat turpis et enim vehicula, in rhoncus mauris ultrices.
+                    Curabitur blandit fermentum eleifend.
+                  </div>
+                </div>
+                <div class="card-footer-basic fluid-x">
+                  <div class="footer-links fluid-x">
+                    <button class="primary-btn-card">READ</button>
+                    <button class="primary-btn-card">BOOKMARK</button>
+                  </div>
+                  <div class="footer-icons fluid-x">
+                    <button class="icon-btn"><i class="far fa-heart"></i></button>
+                    <button class="icon-btn"><i class="fas fa-share-alt"></i></button>
+                    <button class="icon-btn"><i class="fas fa-ellipsis-v"></i></button>
+                  </div>
+                </div>
+                <div class="card-badge">
+                  In progress
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+        <iframe src="https://carbon.now.sh/embed/8HJPxzwsW6W5dJI1htjU" sandbox="allow-scripts allow-same-origin">
+        </iframe>
+        <h2>Holy Grail Layout</h2>
+        <div class="example-container">
+          <div class="grid-holy-grail">
+            <div class="grid-header">
+              Header
+            </div>
+            <div class="grid-sidebar">
+              Sidebar
+            </div>
+            <div class="grid-main">
+              Main
+            </div>
+            <div class="grid-ad">
```

Comment:
Cannot view the ad grid in CSB. Putting it here as it might not be loading on some specific browsers.

Reply:
hey, it seems to be fine in my csb